README.md

```diff
@@ -1,83 +1,304 @@
-* Can create functions with return types.
-* Is familiar with the String, Int, Double, and Bool type.
-* Can perform arithmetic operations on Int and Double.
-* Understands if and else clause statements.
-* Can create and use Arrays.
-* Can iterate over an Array using a for-in loop.
-* Can iterate over an Array calling enumerate().
-* Work with the following methods on arrays:
-	* append()
-	* insert(_:atIndex:)
-	* removeAtIndex()
-	* subscript syntax with arrays
-	* count
-	* isEmpty
-	* Optionals
-
-## Outline/ Notes
-
-* Start with the known- students know how to iterate over arrays. Show an example of iterating over an array. 
-* Before diving into an explicit definition of what a tuple is, it's important for the student to understand the problem they are solving. This can be done by providing a real world example/metaphor they can relate to... or if it's showing them code, it's showing them something they are aware of first.
-
-```swift
-let myFavoriteStarWarsFilms = [
-    "Star Wars - Episode I",
-    "Star Wars - Episode II",
-    "Star Wars - Episode III"
-]
+## Introduction
+You already know that an array is a list of values, and in an array, all of the values have to be of a certain type, i.e. if you have an array of strings, it cannot contain integers.
+
+A tuple is also a list of values, but a tuple's values can be of different types. Also, unlike an array, you can't add values to a tuple once it's defined.
+
+The values in a tuple are generally related to one another. For example, you might have a tuple with a person's name and their age. In this regard, rather than call a tuple a _list_ of values, it would be more accurate to define a tuple as a _group of related values_.
+
+Enough theory, let's dive into some examples!
+
+## Tuples vs Arrays
+
+Did you skip breakfast this morning? Don't worry I have a great assortment of `breakfastBeverages` for you, which should tide you over until your coffee break later in the lesson. :)
+
+We'll start off comparing tuples with arrays. Consider the following array:
+
+```swift
+var breakfastBeverages = ["Milk", "Coffee", "Orange Juice"]
+```
+
+You can add an item to an array, but you can't add anything to a tuple.
+
+```swift
+breakfastBeverages.append("Apple Juice") // ["Milk", "Coffee", "Orange Juice", "Apple Juice"]
+```
+
+An array is a great data structure to use for a list of items of a certain type. Now let's consider a case in which an array could be used, but it wouldn't really be appropriate to do so.
+
+Let's say we wanted to store some properties of a beverage, like whether we prefer a certain beverage hot or cold. We'd need to store the beverage name and either "Hot" or "Cold".
+
+Since both items we want to store are strings, we could technically store them in an array. However what would it be an array of? Beverages? Temperatures?
+
+```swift
+let temperatureArray = ["Coffee", "Hot"]
+let beverageName = temperatureArray[0]
+let beverageTemperature = temperatureArray[1]
+```
```

Comment:
I like this example a lot.

To reiterate the format you're taking here (which I like), we like to get the student in a frame of mind.. that has them seeing the problem they are about to solve before showing them what the syntax is and what a formal definition is.

Here's an example I came up with about Classes (might be silly, wrote it up quick.. but you get the point).

--- 

In prior lessons, we've created multiple variables which represented the planets. But.. these variables were of type `String`. We were only able to store the various String Literal values (i.e. "Earth", "Mars", etc.) in these variables. But what about all of the other attributes that make up a planet? How do we deal with those?

```swift
let name = "Earth"
let numberOfMoons = 1
var percentOxygen = 0.21
var hasLife = true
```
What if we wanted to create every single Planet like this? We would have variables floating around everywhere.
Can create a new type (name it whatever we like) and store **ALL** of these variables/constants above into **ONE** variable that represent the various attributes that make up a real life planet?
We can create our own types. `Class`

```swift
class Planet {
    //TODO: Write code here
}
```

We just created a new type. Just like `String`, `Bool`, and `Int` are types, `Planet` is now a type. You're an expert now:

![Neil](https://media.giphy.com/media/2XsHRy5G950E47BVuww/giphy.gif)